main.cpp

```diff
@@ -226,17 +178,18 @@ struct Numeric<double>
                 std::cout << "Dividing by 0!" << std::endl;
             }
         }
-        
-        *value /= myNumber; 
+
```

Comment:
I have recently updated these instructions in Part7 accordingly: 
```
#11) now that your class is templated, you'll need to adjust your logic in your division function to handle if your input is a zero or not based on your templated type.  
        - look up how to use std::is_same<>::value on cppreference to determine the type of your template parameter.
        
        - look up how to use std::numeric_limits<>::epsilon() to determine if you're dividing by a floating point 0

        - in plain-english, you'll need to implement logic like this:
        if your template type is an int
                if your parameter's type is also an int
                        if your parameter is 0
                                don't do the division
                else if it's less than epsilon
                        dont do the divison
        else if it's less than epsilon
                warn about doing the division

        - to make these checks work during compilation, your if() statements will need to be 'constexpr':  if constexpr (expression)
```
- [ ] revise your logic here.